src/google/protobuf/unittest_proto3.proto

```diff
@@ -140,6 +140,11 @@ message TestAllTypes {
     string oneof_string = 113;
     bytes oneof_bytes = 114;
   }
+  
+  // Sample maps
```

Comment:
I'd prefer to use the same .proto file as c++

https://github.com/google/protobuf/blob/master/src/google/protobuf/map_unittest.proto